dotnet/src/Connectors/Connectors.AzureOpenAI/Services/AzureOpenAITextToImageService.cs

```diff
@@ -46,11 +46,8 @@ public AzureOpenAITextToImageService(
     {
         Verify.NotNullOrWhiteSpace(apiKey);
 
-        var connectorEndpoint = !string.IsNullOrWhiteSpace(endpoint) ? endpoint! : httpClient?.BaseAddress?.AbsoluteUri;
-        if (connectorEndpoint is null)
-        {
-            throw new ArgumentException($"The {nameof(httpClient)}.{nameof(HttpClient.BaseAddress)} and {nameof(endpoint)} are both null or empty. Please ensure at least one is provided.");
-        }
+        var connectorEndpoint = (!string.IsNullOrWhiteSpace(endpoint) ? endpoint! : httpClient?.BaseAddress?.AbsoluteUri)
```

Comment:
nit: this change saves 5 characters in the 3 lines at the cost of readability. I'm fine with the change, but it takes me a little bit longer to read the code than before.